serializer/pegnodes/src/pegnodes/tests/tests.clj

```diff
@@ -118,6 +118,16 @@
                 (subs actual-str idx (min (count actual-str) (+ idx 80)))
                 "\033[0m\n")))))
 
+(defn ensure-ids-are-equiv
+  "ensure, the ids point to pegs that form a structual bijection"
+
+  ([expected-id actual-id expected-string actual-string] (ensure-ids-are-equiv expected-id actual-id nil))
+  ([expected-id actual-id expected-string actual-string msg]
+   (let [bijection (PegNode/isStructuralBijection expected-id actual-id)]
+      (if bijection
+          (clojure.test/is bijection "fail")
+          (ensure-strings-are-same expected-string actual-string msg)))))
```

Comment:
Yeah this was because I didn't have a great idea off the top of my head for how to get isStructuralBijection to produce helpful debugging data. We should probably address that since the string-based debugging information actually tends to be pretty useless when it comes to theta-related bugs. I think the natural thing to do would be to wrap have a wrapper for isStructuralBijection that returns a string containing debug info on failure. Any ideas on how to maybe structure that output?